extensions.json

```
@@ -883,6 +883,12 @@
       "repository": "https://github.com/mitchdenny/ecdc",
       "version": "1.4.0"
     },
+    {
+      "id": "mongodb.mongodb-vscode",
+      "repository": "https://github.com/mongodb-js/vscode",
+      "version": "0.4.0",
+      "checkout": "v0.4.0"
```

Same remark as in https://github.com/open-vsx/publish-extensions/pull/275#discussion_r567922516: 

```suggestion
      "repository": "https://github.com/mongodb-js/vscode/releases/download/v0.4.0/mongodb-vscode-0.4.0.vsix",
      "version": "0.4.0"
```